runner/serializers.py

```diff
@@ -140,7 +140,7 @@ def get_status_url(self, obj):
 
     class Meta:
         model = Run
-        fields = '__all__'
+        fields = ('id', 'name', 'status', 'tags', 'app', 'inputs', 'outputs', 'status_url', 'created_date', 'job_statuses','execution_id','output_metadata','output_directory','operator_run','job_group','notify_for_outputs','finished_date')
```

Comment:
`message` should be included in RunSerializerFull as well, since we are currently not able to see the message when we do a GET /v0/run/api. I don't think it is worth hotfixing, but lets make the change for 1.9.0.